src/includes/classes/Plugin.php

```diff
@@ -490,6 +490,12 @@ public function setup()
         /*[/pro]*/
 
         /*[pro strip-from="lite"]*/
+        if ($this->options['when_logged_in'] === '1' && $this->applyWpFilters(GLOBAL_NS.'_when_logged_in_no_admin_bar', true)) {
+          show_admin_bar(FALSE); // Prevent admin bar from being cached.
```

Comment:
@jaswsinc Ah, good to know. Cool. In that case `show_admin_bar()` should be fine here. :-)